components/TweetCarouselDelimiter.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
+    <v-list class="tweet-carousel-delimiter">
+      <v-list-item
+        v-for="(n, i) in steps"
```

Comment:
nit: could be more descriptive vars. Not sure what `n` is.

Reply:
Sure - let me change it to something like `_` or `unsused`, as it's well, unused =). In case you're curious, if you give `v-for` an int to iterate, it will either give you back a) a single 1-indexed ordinal number (`n`), or b) a tuple of that same 1-indexed value, and a 0-indexed index value (`(n, i)`).

I think `i` is fine here since it's a pretty widely used paradigm.

Reply:
tbh, I actually know what it is 😄 since I've woorked a bunch in vue, but I believe it's better to use descriptive vars for other people reading the code to easily understand without having to research every param. The exception to this is when it's obvious what the var is and it's created within the scope of the function, not a param. loop variables are an example. 

Careful with `_` since there is the underscore library. 😸

Reply:
I would just use vue's names used in their [example](https://v3.vuejs.org/guide/list.html#mapping-an-array-to-elements-with-v-for) `(item, index)`

Reply:
done, but used `idx` instead, as index is already a used keyword in this component

Reply:
For that reason I actually think `i` is better, but I won't fight it.

Reply:
honestly, `i` is not so bad since it's consistently just an iterator var. I think `n` is the lesser known one.